.github/infrastructure/conformance/azure/setup-azure-conf-test.sh

```diff
@@ -213,11 +213,11 @@ IOT_HUB_EVENT_HUB_CONNECTION_STRING_VAR_NAME="AzureIotHubEventHubConnectionStrin
 IOT_HUB_BINDINGS_CONSUMER_GROUP_VAR_NAME="AzureIotHubBindingsConsumerGroup"
 IOT_HUB_PUBSUB_CONSUMER_GROUP_VAR_NAME="AzureIotHubPubsubConsumerGroup"
 
-KEYVAULT_CERT_NAME="AzureKeyVaultSecretStoreCert"
-KEYVAULT_CLIENT_ID_VAR_NAME="AzureKeyVaultSecretStoreClientId"
-KEYVAULT_SERVICE_PRINCIPAL_CLIENT_SECRET_VAR_NAME="AzureKeyVaultSecretStoreServicePrincipalClientSecret"
-KEYVAULT_SERVICE_PRINCIPAL_CLIENT_ID_VAR_NAME="AzureKeyVaultSecretStoreServicePrincipalClientId"
-KEYVAULT_TENANT_ID_VAR_NAME="AzureKeyVaultSecretStoreTenantId"
+KEYVAULT_CERT_NAME="AzureKeyVaultCert"
```

Comment:
Reviewers: I've manually copied over the secrets/certs, so they exist with both names in the Key Vault (for both old and new tests)

Reply:
Will they auto-expire or should we have something to remind us to remove them in the future?

Reply:
It will expire but this is not a new thing for this PR. Just re-using the key that was there before.

We will need to rotate the keys in the future. The reminder will be that the CI will fail 🤣

Reply:
LGTM